src/app/shared/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,3 +1,7 @@
 <div class="content">
-  Courses
+  <ul>
+    <li *ngFor="let item of breadcrumbs; last as last">
+      {{item}} <span class="slash">{{last ? '' : '/'}}</span>
```

Comment:
This slash could be added with styles only

Reply:
done